src/types/schema.ts

```diff
@@ -72,14 +72,15 @@ export interface Interest {
 }
 
 // irrelevant until mvp
-export interface LimitedAssistance {
+export interface LimitedCaseAssignment {
```

Comment:
could we update the LCA schema to match this schema in this screenshot:
<img width="229" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/c396f053-80f1-4684-a7e2-3849299b6a8b">